pod/perlfunc.pod

```diff
@@ -6686,6 +6686,14 @@ X<read> X<file, read>
 
 =for Pod::Functions fixed-length buffered input from a filehandle
 
+    open(my $FH, "<", "input.txt") or die("Cannot open file");
+
+    my $buf = "";
+    my $num = 0;
+
+    $num = read($FH, $buf, 32);       # Read the first 32 bytes
+    $num = read($FH, $buf, 16, 1024); # Read 16 bytes at position 1024
```

Comment:
position 1024 in the file?  That was my default reading of the comment.

It's actually position 1024 in `$buf`, but if someone reads the example and doesn't carefully read the rest I could see this causing confusion.

Reply:
Indeed, simply changing that to say "into position 1024" would be clearer